components/brave_rewards/resources/ui/components/donate/index.tsx

```diff
@@ -124,12 +147,15 @@ export default class Donate extends React.PureComponent<Props, State> {
           <StyledDonationTitle isMobile={isMobile}>{title}</StyledDonationTitle>
             <StyledAmountsWrapper isMobile={isMobile}>
               {
-                donationAmounts && donationAmounts.map((donation: Donation) => {
+                donationAmounts && donationAmounts.map((donation: Donation, index: number) => {
+                  const isCurrentAmount = donation.tokens === currentAmount.toString()
+                  const isDefaultAmount = index === 0 && !this.state.amountChanged
```

Comment:
I talked with @davidtemkin and he is ok with going with a middle selection. So let's change index here from 0 to 1.

Reply:
done